examples/emberjs/js/controllers/todo_controller.js

```diff
@@ -33,23 +34,25 @@
 				// Re-set our newly edited title to persist its trimmed version
 				this.set('bufferedTitle', bufferedTitle);
 				this.set('isEditing', false);
+                return false;
 			},
 
 			cancelEditing: function () {
 				this.set('bufferedTitle', this.get('title'));
 				this.set('isEditing', false);
+                return false;
 			},
 
 			removeTodo: function () {
 				this.removeTodo();
+                return false;
```

Comment:
Actually, I was wrong. Event bubbling is disabled by default for actions, so these `return false;` statements are definitely unnecessary.

From http://emberjs.com/api/classes/Ember.ObjectController.html#toc_bubbling:

> By default, an action will stop bubbling once a handler defined on the actions hash handles it. To continue bubbling the action, you must return true from the handler